resources/js/notifications/components/NotificationsList.vue

```diff
@@ -1,167 +1,252 @@
-                    });
-            }
-        }
+        {
+          title: () => this.$t("Message"),
+          name: "__slot:subject",
+          sortField: "subject",
+        },
+        {
+          title: () => this.$t("Time"),
+          name: "created_at",
+          sortField: "created_at",
+        },
+      ],
     };
+  },
+  computed: {
+    url() {
+      return this.notification.data?.url;
+    },
+    isComment() {
+      return this.data.type === "COMMENT";
+    },
+  },
+  mounted: function mounted() {
+    const params = new URL(document.location).searchParams;
+    const successRouting = params.get("successfulRouting") === "true";
+    if (successRouting) {
+      ProcessMaker.alert(this.$t("The request was completed."), "success");
+    }
+  },
+  methods: {
+    read(id) {
+      ProcessMaker.removeNotifications([id]).then(() => {
+        this.fetch();
+      });
+    },
+
+    unread(id) {
+      ProcessMaker.unreadNotifications([id]).then(() => {
+        this.fetch();
+      });
+    },
+
+    getSortParam() {
+      if (this.sortOrder instanceof Array && this.sortOrder.length > 0) {
+        return `&order_by=${this.sortOrder[0].sortField
+        }&order_direction=${this.sortOrder[0].direction}`;
+      }
+      return "";
+    },
+
+    transform(data) {
+      // eslint-disable-next-line no-restricted-syntax
+      for (const record of data.data) {
+        record.created_at = this.formatDate(record.created_at);
+        if (record.read_at) {
+          record.read_at = this.formatDate(record.read_at);
+        } else {
+          record.read_at = null;
+        }
+      }
+      return data;
+    },
+
+    formatDate(dateTime) {
+      const months = [
+        "January", "February", "March", "April", "May", "June",
+        "July", "August", "September", "October", "November", "December",
+      ];
+
+      const dateObj = new Date(dateTime);
+      const currentDate = new Date();
+
+      if (
+        dateObj.getDate() === currentDate.getDate()
+        && dateObj.getMonth() === currentDate.getMonth()
+        && dateObj.getFullYear() === currentDate.getFullYear()
+      ) {
+        const hours = dateObj.getHours();
+        const minutes = dateObj.getMinutes();
+        return `${this.addLeadingZero(hours)}:${this.addLeadingZero(minutes)}`;
+      }
+      const month = dateObj.getMonth();
+      const day = dateObj.getDate();
+      const formattedMonth = months[month];
+      return `${formattedMonth} ${day}`;
+    },
+
+    addLeadingZero(value) {
+      return value < 10 ? `0${value}` : value;
+    },
+
+    fetch() {
+      this.loading = true;
+      if (this.cancelToken) {
+        this.cancelToken();
+        this.cancelToken = null;
+      }
+      const { CancelToken } = ProcessMaker.apiClient;
+
+      // Load from your API client (adjust the API endpoint and parameters as needed)
+      ProcessMaker.apiClient
+        .get(
+          `notifications?page=${
```

Comment:
@pmoskrojas All the rows with users have "Unknown User" I think you need to add `include=user` to the query to get the additional user info in the request